applications/areg-portal/frontend/src/components/Home/HomeHeader.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { AppBar, Box, Container, Grid, Stack, Typography } from "@mui/material";
+import StyledButton from "../StyledButton";
+import { AddAntibodyIcon, DownloadIcon } from "../icons";
+import TableToolbar from "./TableToolbar";
+
+const HomeHeader = () => {
+  return (
+    <Box>
+      <AppBar elevation={0} sx={{ top: "4.5rem" }}>
+        <Container maxWidth="xl">
+          <Stack direction="column" spacing={1.5} mb={2}>
+            <Box
+              display="flex"
+              height="4.125rem"
```

Comment:
Really need to set the height of this? Setting the height is always dangerous as the content may change at some point and may start behaving wrong.

Reply:
done